bin/check-windows-process.rb

```
@@ -62,7 +62,7 @@
 parser.parse!
 
 if options[:procname] == ''
-  STDERR.puts 'Expected a process to match against.'
+  warn 'Expected a process to match against.'
```

if we warn here the exception will be swallowed.